### ISSUE TITLE
"unexpected indent" if first line of module is a class (v2.0.3)

### ISSUE THREAD

**Issue by collinstocks-oscar** [rubik/radon#133]
`radon cc --json` fails on the following file (assuming no blank lines above the class):

```python
class Foo(object):
    pass
```

```
$ radon cc --json /tmp/foo.py
{"/tmp/foo.py": {"error": "unexpected indent (<unknown>, line 1)"}}
```

Obviously there's no real error here. Any idea why this might be?

```
$ pip freeze | grep radon
radon==2.0.3
```

**Comment by rubik** [comment]
Thanks for the report. I see that this error happens only in Python 2, whereas in Python 3 it works correctly. It's a weird one, I'll look into it.

**Comment by collinstocks-oscar** [comment]
Thanks! This is a rare error since usually the first line is either a comment or an import statement, so I'm not personally in a rush to see a fix. But like you said, it's probably related to the encoding handling (if it's a new error -- I didn't test on earlier versions of radon).

**Comment by rubik** [comment]
It should be fixed in version 2.1.1 which I pushed to PyPI.
